src/register.js

```diff
@@ -0,0 +1,34 @@
+export function findProduct(items, code) {
+    for(let i = 0; i < items.length; i++) {
+        const item = items[i];
+        if(item.code === code) {
+            return item;
+        }
+    }
+    return null;
+}
+    
+export function getLineTotal(quantity, price) {
+    return Math.round(quantity * price * 100) / 100;
```

Comment:
not sure why you did round. but i think youd get the same exact output if you did
return quantity * price;